signature-specification.md

```diff
@@ -1,243 +1,118 @@
 
 ## Signature Envelope
 
 The Signature Envelope is a standard data structure for creating a signed message.
 A signature envelope consists of the following components:
 
-- Payload `m`: The data that is integrity protected - e.g. descriptor of the artifact being signed.
-- Signed attributes `v`: The signature metadata that is integrity protected - e.g. signature expiration time, signing time, etc.
-- Unsigned attributes `u`: This OPTIONAL property represents signature metadata that is not integrity protected - e.g. timestamp, certificates, etc.
+- Payload/Message `m`: The data that is integrity protected - e.g. descriptor of the artifact being signed.
+- Signed attributes `v`: The signature metadata that is integrity protected - e.g. signature expiration time, creation time, etc.
+- Unsigned attributes `u`: Unsigned attributes `u`: These attributes are not signed by the signing key that generates the signature. We anticipate unsigned attributes contain content that may be signed by an different party e.g. Certificate chain signed by a CA, or TSA countersignature signed by the TSA.
 - Cryptographic signatures `s`: The digital signatures computed on payload and signed attributes.
 
 A signature envelope is `e = {m, v, u, s}` where `s` is signature.
 
-Notary v2 supports [JWS JSON Serialization](https://datatracker.ietf.org/doc/html/rfc7515) as signature envelope format with some additional constraints but makes provisions to support additional signature envelope format.
+This specification defines the set of signed and unsigned attributes that make up a valid The Notary v2 signature. This specification aims to be be agnostic of signature envelope format (e.g. COSE, JWS), details of encoding the envelope in a specific signature envelope format are covered in in separate specs.
+
+Notary v2 supports the following envelope formats:
+
+- [JWS](./signature-envelope-jws.md)
 
 ### Payload
 
-Notary v2 requires Payload to be the content **descriptor** of the subject manifest that is being signed.
+Notary v2 payload is a JSON document with media type `application/vnd.cncf.notary.payload.v1+json` and has following properties.
 
-1. Descriptor MUST contain `mediaType`, `digest`, `size` fields.
-1. Descriptor MAY contain `annotations` and if present it MUST follow the [annotation rules](https://github.com/opencontainers/image-spec/blob/main/annotations.md#rules). Notary v2 uses annotations for storing both Notary specific and user defined signed attributes. The prefix `io.cncf.notary` in annotation keys is reserved for use in Notary v2 and MUST NOT be used outside this specification.
-1. Descriptor MAY contain `artifactType` field for artifact manifests, or the `config.mediaType` for `oci.image` based manifests.
+- `subject` : Required property whose value is the **OCI descriptor** of the subject manifest that is being signed, with media type of `application/vnd.cncf.oras.artifact.descriptor.v1+json`.
```

Comment:
> application/vnd.cncf.oras.artifact.descriptor.v1+json

The examples below have mediaType of the artifact that needs to be signed. Can you clarify (for my understanding), why we are saying the mediaType should be `application/vnd.cncf.oras.artifact.descriptor.v1+json` here?

Reply:
I actually removed the media type, we support both [OCI descriptor](https://github.com/opencontainers/image-spec/blob/main/descriptor.md) and ORAS [artifact descriptors](https://github.com/oras-project/artifacts-spec/blob/main/descriptor.md) in the payload.